.github/workflows/ci.yaml

```diff
@@ -1,74 +1,51 @@
-name: Fmt and Clippy
-
-# Run 'rustfmt' and 'clippy' linting.
+name: Build and test
```

Comment:
Are we actually building here? Are these changes related to the rust upgrade or is it just a simplification?